### PR TITLE
Add additional files to repo for

### PR DESCRIPTION
**Pull request by GitKrakenQA**
Added additional files to help with testing for this repository. This should allow internal devs and QA to generate PRs with more edge cases since there will be more files to work with.

### CODE REVIEW

File: blockOfText.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ We’re all in a glow—
 Nell, Dolly, and Willie, and Dan;
 For the primest of fun,
 When all’s said and done,
+//TODO
```
Comment (MannyGrill) [inline blockOfText.md:24]: Please remove TODOs

File: src/important.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+App.factory('rickAstley', function() {
+  var messages = [
+    'Never gonna give you up',
+    'Never gonna let you down',
+    'Never gonna run around and desert you',
+    'Never gonna make you cry',
+    'Never gonna say goodbye',
+    'Never gonna tell a lie and hurt you'
+  ];
```
Comment (MannyGrill) [inline src/important.js:9]: 8882bcf327896ab79fb97e85ae63a002

File: src/strings/lines.txt [hunk 3]
```diff
@@ -0,0 +1,1027 @@
...
+lines
+lines
+lines
+lines
+lines
+lines
+lines
+lines
+lines
+lines
```
Comment (MannyGrill) [inline src/strings/lines.txt:1000]: What was the reasoning for putting `lines` instead of `line` here?

File: src/strings/en-us.json [hunk 4]
```diff
@@ -0,0 +1,2580 @@
+{
+  "languageOption": {
+    "label": "English (US)",
```
Comment (MannyGrill) [inline src/strings/en-us.json:3]: Will we add more language support?

**Review by MannyGrill — CHANGES_REQUESTED** [review]

File: blockOfText.md (hunk 1 above)
Comment (GitKrakenQA) [reply blockOfText.md:24]: Oops nice catch!

File: src/strings/en-us.json (hunk 4 above)
Comment (GitKrakenQA) [reply src/strings/en-us.json:3]: Yes

File: blockOfText.md (hunk 1 above)
Comment (MannyGrill) [reply blockOfText.md]: Nice

Comment (MannyGrill) [reply blockOfText.md]: hi

**Review by MannyGrill — APPROVED** [review]
Approved

**Pull request merged**